src/components/media/Media.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/variables';
+
+// the outermost element
+.bdl-Media {
+    align-items: flex-start;
+    display: flex;
+}
+
+// the media content, ie, an avatar
+.bdl-Media-img {
```

Comment:
Each sub-component should have its own top-level class name, such as `.bdl-MediaImage`. It's probably fine for them to live in the same stylesheet, though, as long as re-importing it in each component doesn't create duplicate styles in the output CSS.

Reply:
I was following this 

https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#ComponentName-descendentName

Reply:
That applies when different elements live in the same file, but since these are declared as separate components, they should be named as such.

Reply:
The sub-elements are meant to be used like `Media.Body` not as separate imports.

These are sub-elements almost exactly like the Tweet example in the SUIT docs.